code/modules/antagonists/changeling/powers/fleshmend.dm

```diff
@@ -14,6 +14,9 @@
 	if(user.has_status_effect(STATUS_EFFECT_FLESHMEND))
 		to_chat(user, "<span class='warning'>Our healing's effectiveness is reduced \
 			by quick repeated use!</span>")
+	if(user.bodytemperature + 50 <= user.dna.species.body_temperature)
+		to_chat(user, "<span class='warning'>Our healing's effectiveness is reduced \
+			by our cold body!</span>")
```

Comment:
Can you move this to the `on_apply`? Perhaps even make the check into a proc so that you only do the logic once

Reply:
I groaned when I saw these requests when they sounded complex but they were not, yay!